hash/mac_hdflop.xml

```diff
@@ -5,125 +5,3347 @@ license:CC0
+			</dataarea>
+		</part>
+		<part name="flop10" interface="floppy_3_5">
+			<feature name="part_id" value="AppleScript Setup" />
+			<dataarea name="flop" size="1474560">
+				<rom name="AppleScript Setup.img" size="1474560" crc="2a049c49" sha1="8f9d9946e005bc1abe37636156827b6af74571fb" />
+			</dataarea>
+		</part>
+		<part name="flop11" interface="floppy_3_5">
+			<feature name="part_id" value="Before You Install" />
+			<dataarea name="flop" size="819200">
+				<rom name="Before You Install.img" size="819200" crc="f6d55a1c" sha1="81906cd5519f02a1ae4d9c074bd432a2f869edfc" />
+			</dataarea>
+		</part>
+		<part name="flop12" interface="floppy_3_5">
+			<feature name="part_id" value="Disk Tools" />
+			<dataarea name="flop" size="1474560">
+				<rom name="Disk Tools.img" size="1474560" crc="23adcd0f" sha1="5f81b98c6adb47c05d5e16fc5f8c84c5a2dfcb34" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="sys711pb150" supported="no">
+		<description>System Software 7.1.1 (PowerBook 150, US English)</description>
+		<year>1994</year>
+		<publisher>Apple</publisher>
+		<part name="flop1" interface="floppy_3_5">
+			<feature name="part_id" value="Install Me First" />
+			<dataarea name="flop" size="1474560">
+				<rom name="InstallMeFirst.img" size="1474560" crc="977805c8" sha1="5171b008ec41954d3e0204ce2e4522f4d60defa2" />
+			</dataarea>
+		</part>
+		<part name="flop2" interface="floppy_3_5">
+			<feature name="part_id" value="Disk Tools" />
+			<dataarea name="flop" size="1474560">
+				<rom name="Disk Tools.img" size="1474560" crc="3ed3c068" sha1="48aa1362943a775f00e54e9ee05a773f563e6406" />
+			</dataarea>
+		</part>
+		<part name="flop3" interface="floppy_3_5">
+			<feature name="part_id" value="Goodies" />
+			<dataarea name="flop" size="1474560">
+				<rom name="Goodies.img" size="1474560" crc="1f6c79e0" sha1="f2e8ab8b0fc337bebd44eb242237db1a060a0edf" />
+			</dataarea>
+		</part>
+		<part name="flop4" interface="floppy_3_5">
+			<feature name="part_id" value="Printing &amp; Fonts" />
+			<dataarea name="flop" size="1474560">
+				<rom name="Printing And Fonts.img" size="1474560" crc="7758c49a" sha1="1456129ffddf67aebd100812e89f16088ba80d15" />
+			</dataarea>
+		</part>
+		<part name="flop5" interface="floppy_3_5">
+			<feature name="part_id" value="Additions" />
+			<dataarea name="flop" size="1474560">
+				<rom name="Additions.img" size="1474560" crc="b7d59e65" sha1="eb6a41f97c762d423a3c0b77e11a340bfaeff5df" />
+			</dataarea>
+		</part>
+		<part name="flop6" interface="floppy_3_5">
+			<feature name="part_id" value="Utilities" />
+			<dataarea name="flop" size="1474560">
+				<rom name="Utilities.img" size="1474560" crc="3cadd68a" sha1="1ecef7d3a19cf44dc8942faeb70c2653aee3ee2b" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="sys712lcq" supported="no">
+		<description>System Software 7.1.2 (LC 580/Quadra 630, US English)</description>
```

Comment:
Was this ever distributed in this form, or are these generated from the Disk Copy images for creating a set of installation floppies on the CD?  Did the LC580 actually come in a configuration without a CD-ROM drive?  I only ever saw the system software on CD, with Disk Copy images for creating a set of floppies if you needed them for some reason.

Reply:
I'm honestly not sure about this, but it's definitely possible. I could add a note indicating that those images are probably extracted from the CD.